Kiosk/Bid Fulfillment/RegistrationConfirmationViewController.swift

```diff
@@ -1,10 +1,17 @@
 import Foundation
 import UIKit
+import RxSwift
 
 let targetConditionsOfSaleText = "Conditions of Sale"
 
 class RegistrationConfirmationViewController: UIViewController {
     @IBOutlet weak var confirmationLabel: ARSerifLabel!
+    @IBOutlet weak var conditionsOfSaleCheckbox: UIButton!
+
+    fileprivate let _conditionsOfSaleChecked: Variable<Bool> = Variable(false)
+    var conditionsOfSaleChecked: Observable<Bool> {
+        return _conditionsOfSaleChecked.asObservable()
+    }
```

Comment:
Why private + getter, and why observable? I think I understand:
- the boolean itself is private and not so important
- the button in the storyboard file down below has an action/attribute to toggle the value (`<action selector="conditionsOfSaleCheckmarkTapped:" destination="q7z-ij-DNu" eventType="touchUpInside" id="gPa-dm-9Dt"/>`)
- somewhere up in `RegisterViewController` the observable gets `conditionsOfSaleChecked.bind(to: self.confirmButton.rx.isEnabled)` which means that if it is checked the button gets enabled.

Reply:
Great question! The critical thing to note is that `Variable` is read/write, while `Observable` is read-only. By declaring the read/write version as `file private`, we make it so only this view controller can set the value. But by declaring a read-only version that others can access, the other controller can bind as you described.